packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -281,122 +32,3 @@ QUnit.module('{{#with}} helper binding to view keyword', {
 QUnit.test('{{with}} helper can bind to keywords with \'as\'', function() {
```

Comment:
@wycats believe that this test could be ported to the new harness although we don't expect it to pass yet (i.e. `@htmlbars ...` for now). The `view` keyword should still work in htmlbars even though we switched the top-level thing to a component, but we could be missing some other pieces?